### DISCUSSION TITLE
FutureWarning: In future versions `DataFrame.to_latex` is expected to utilise the base implementation of `Styler.to_latex`

### DISCUSSION THREAD

**Discussion by alinuman15** [oemof/tespy#386] [General]
Hi All: 
How to get rid of the following warning ?
WARNING:py.warnings:C:\Users\Owner\anaconda3\envs\user1\lib\site-packages\tespy\tools\document_models.py:951: FutureWarning: In future versions `DataFrame.to_latex` is expected to utilise the base implementation of `Styler.to_latex` for formatting and rendering. The arguments signature may therefore change. It is recommended instead to use `DataFrame.style.to_latex` which also contains additional functionality.
  latex = df.to_latex(
Thank you in advance

**Comment by fwitte** [comment]
Dear @alinuman15,

this is a pandas future warning, that the latex export will fail in the future with the current implementation of that feature in TESPy. I am going to transfer this to an issue, since it should be fixed in TESPy. What you can do at the moment without changing things in the TESPy back-end:

- Follow the instructions in the most upvoted answer here: https://stackoverflow.com/questions/15777951/how-to-suppress-pandas-future-warning
- Downgrade your pandas version to a version that does not have this future warning, e.g. the minimum required version of pandas in the TESPy dependencies

I'll try to fix it on the TESPy side, once I have the time for that.

**Reply by alinuman15** [reply]
Thanks for prompt replying. Please, inform me back when updating that issue.
